core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java

```diff
@@ -79,20 +84,20 @@ public final void initialize(TelemetryConfiguration configuration) {
         try {
             String configurationFile = new ConfigurationFileLocator(CONFIG_FILE_NAME).getConfigurationFile();
             if (Strings.isNullOrEmpty(configurationFile)) {
-                configuration.setChannel(new InProcessTelemetryChannel());
+                initializeMinimumCapabilities(null, configuration);
                 return;
             }
 
             ApplicationInsightsXmlConfiguration applicationInsightsConfig = builder.build(configurationFile);
             if (applicationInsightsConfig == null) {
                 InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.ERROR, "Failed to read configuration file");
-                configuration.setChannel(new InProcessTelemetryChannel());
+                initializeMinimumCapabilities(applicationInsightsConfig, configuration);
```

Comment:
In this case `applicationInsightsConfig == null` so why not passing to `initializeMinimumCapabilities` null explicitly?